Build failure with CFE_PLATFORM_TIME_CFG_SRC_TIME == true

**Describe the bug**
Build failures when enabling time source configuration.

```
/cfe/modules/time/fsw/src/cfe_time_tone.c:570:61: error: ‘CFE_TIME_ToneDataCmd_t {aka struct CFE_TIME_ToneDataCmd}’ has no member named ‘CmdHeader’; did you mean ‘CommandHeader’?
             CFE_SB_TransmitMsg(&CFE_TIME_Global.ToneDataCmd.CmdHeader.Msg, false);
```

**To Reproduce**
- Set `CFE_PLATFORM_TIME_CFG_SOURCE` true
- Set `CFE_PLATFORM_TIME_CFG_SRC_TIME` true
- Set `CFE_MISSION_TIME_CFG_FAKE_TONE` false

**Expected behavior**
Should build and run.

**Code snips**
See warning above, note 3 instances of the old reference.

**System observed on:**
 - Hardware: i5/wsl
 - OS: Ubuntu 18.04
 - Versions: Bundle main

**Additional context**
None

**Reporter Info**
Jacob Hageman - NASA/GSFC

--- a/modules/time/fsw/src/cfe_time_tone.c
+++ b/modules/time/fsw/src/cfe_time_tone.c
@@ -283,7 +283,7 @@ int32 CFE_TIME_ToneSendMET(CFE_TIME_SysTime_t NewMET)
             /*
             ** Send "time at the tone" command data packet...
             */
-            CFE_SB_TransmitMsg(&CFE_TIME_Global.ToneDataCmd.CmdHeader.Msg, false);
+            CFE_SB_TransmitMsg(&CFE_TIME_Global.ToneDataCmd.CommandHeader.Msg, false);
 
             /*
             ** Count of "time at the tone" commands sent with external data...
@@ -425,7 +425,7 @@ int32 CFE_TIME_ToneSendGPS(CFE_TIME_SysTime_t NewTime, int16 NewLeaps)
             /*
             ** Send "time at the tone" command data packet...
             */
-            CFE_SB_TransmitMsg(&CFE_TIME_Global.ToneDataCmd.CmdHeader.Msg, false);
+            CFE_SB_TransmitMsg(&CFE_TIME_Global.ToneDataCmd.CommandHeader.Msg, false);
 
             /*
             ** Count of "time at the tone" commands sent with external data...
@@ -567,7 +567,7 @@ int32 CFE_TIME_ToneSendTime(CFE_TIME_SysTime_t NewTime)
             /*
             ** Send "time at the tone" command data packet...
             */
-            CFE_SB_TransmitMsg(&CFE_TIME_Global.ToneDataCmd.CmdHeader.Msg, false);
+            CFE_SB_TransmitMsg(&CFE_TIME_Global.ToneDataCmd.CommandHeader.Msg, false);
 
             /*
             ** Count of "time at the tone" commands sent with external data...

